cdap-api/src/main/java/co/cask/cdap/api/retry/RetriesExhaustedException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.retry;
+
+/**
+ * An exception that indicates an operation failed after it was retried.
+ */
+public class RetriesExhaustedException extends RetryableException {
```

Comment:
yeah I chose 'retryable' since there are other coding projects that use that spelling. Spring has a 'retryable' annotation and there are python, ruby, and javascript projects named 'retryable'.